oag/ja-jp/content/topics/access-gateway/command-line-console-reference-network.htm

```diff
@@ -424,8 +424,8 @@ <h4>プロキシの設定</h4>
                                                             <li value="5">必要に応じてプロキシのユーザー名を入力し、<span class="uicontrol">Enter</span>を押します。 </li>
                                                             <li value="6">必要に応じてプロキシのパスワードを入力し、<span class="uicontrol">Enter</span>を押します。<br /></li>
                                                             <li value="7">プロキシをバイパスする必要があるホスト名を入力します（カンマ区切り）。</li>
-                                                            <li value="8"><span class="uicontrol">y</span>を押してプロキシ設定を確定するか、<span class="uicontrol">N</span>を押して処理を中断します。</li>
-                                                            <li value="9"><span class="uicontrol">改行キー</span>を押してプロキシメニューに戻り、<span class="uicontrol">改行キー</span>を再度押してネットワーキングのメインメニューに戻ります。</li>
+                                                            <li value="8">プロキシ設定を確定する場合は<span class="uicontrol">y</span>、処理を中断する場合は<span class="uicontrol">N</span>を押します。</li>
+                                                            <li value="9"><span class="uicontrol">Enter</span>を押してプロキシメニューに戻り、もう一度<span class="uicontrol">Enter</span>を押して<span class="uicontrol">［Networking（ネットワーキング）］</span>のメインメニューに戻ります。</li>
```

Comment:
@pengyao-okta Previously UI label was JA only, the translator changed it to EN only.